jwst/lib/basic_utils.py

```diff
@@ -2,6 +2,41 @@
 import re
 
 
+def bytes2human(n):
+    """Convert bytes to human-readable format
+
+    Taken from the `psutil` library which references
+    http://code.activestate.com/recipes/578019
+
+    Parameters
+    ----------
+    n : int
+        Number to convert
+
+    Returns
+    -------
+    readable : str
+        A string with units attached.
+
+    Examples
+    --------
+    >>> bytes2human(10000)
+        '9.8K'
+
+    >>> bytes2human(100001221)
+        '95.4M'
+    """
+    symbols = ('K', 'M', 'G', 'T', 'P', 'E', 'Z', 'Y')
```

Comment:
Wow, you really made this future proof! Can't wait to have enough RAM to utilize "Y"!

Reply:
Wonder if Amazon can provide that. Wonder what the cost would be!